redash/models/__init__.py

```diff
@@ -1099,6 +1099,7 @@ class Dashboard(ChangeTrackingMixin, TimestampMixin, BelongsToOrgMixin, db.Model
     tags = Column(
         "tags", MutableList.as_mutable(postgresql.ARRAY(db.Unicode)), nullable=True
     )
+    options = Column(db.Text)
```

Comment:
@rafawendel I think the error is because this is not reflecting the migration, maybe make it like https://github.com/getredash/redash/blob/1b8ff8e810aaa81c8aa8f28aef9e1ca3632b8521/redash%2Fmodels%2F__init__.py#L496 and then retrigger the script to generate it?